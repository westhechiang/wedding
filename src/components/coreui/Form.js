import styled from 'styled-components';
import { theme } from '../../ui';

const Form = styled.form`
  padding: ${props => props.padding || theme.gutter};
  width: ${props => props.width || 'auto'};
`;

export default Form;
