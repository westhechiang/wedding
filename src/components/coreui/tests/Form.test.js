import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Form from '../../../components/coreui/Form';

describe('<Form />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<Form />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
