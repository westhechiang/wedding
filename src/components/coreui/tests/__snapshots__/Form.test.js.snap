// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should match its empty snapshot 1`] = `
<form
  className="sc-bdVaJa kVRThe"
/>
`;
